lib/ansible/modules/cloud/oneandone/oneandone_server.py

```diff
@@ -0,0 +1,644 @@
+        if server:
+            servers.append(server)
+
+    changed = False
+
+    if servers:
+        servers = [_insert_network_data(_server) for _server in servers]
+        changed = True
+
+    return (changed, servers)
+
+
+def remove_server(module, oneandone_conn):
+    """
+    Removes a server.
+
+    module : AnsibleModule object
+    oneandone_conn: authenticated oneandone object.
+
+    Returns a dictionary containing a 'changed' attribute indicating whether
+    the server was removed, and a 'removed_server' attribute with
+    the removed server's hostname and id.
+    """
+    server_id = module.params.get('server')
+    wait = module.params.get('wait')
+    wait_timeout = module.params.get('wait_timeout')
+
+    changed = False
+    removed_server = None
+
+    server = get_server(oneandone_conn, server_id, True)
+    if server:
+        try:
+            oneandone_conn.delete_server(server_id=server['id'])
+            if wait:
+                wait_for_resource_deletion_completion(oneandone_conn,
+                                                      OneAndOneResources.server,
+                                                      server['id'],
+                                                      wait_timeout)
+            changed = True
+        except Exception as ex:
+            module.fail_json(
+                msg="failed to terminate the server: %s" % str(ex))
+
+        removed_server = {
+            'id': server['id'],
+            'hostname': server['name']
+        }
+
+    return (changed, removed_server)
+
+
+def startstop_server(module, oneandone_conn):
+    """
+    Starts or Stops a server.
+
+    module : AnsibleModule object
+    oneandone_conn: authenticated oneandone object.
+
+    Returns a dictionary with a 'changed' attribute indicating whether
+    anything has changed for the server as a result of this function
+    being run, and a 'server' attribute with basic information for
+    the server.
+    """
+    state = module.params.get('state')
+    server_id = module.params.get('server')
+    wait = module.params.get('wait')
+    wait_timeout = module.params.get('wait_timeout')
+
+    changed = False
+
+    # Resolve server
+    server = get_server(oneandone_conn, server_id, True)
+    if server:
+        # Attempt to change the server state, only if it's not already there
+        # or on its way.
+        try:
+            if state == 'stopped' and server['status']['state'] == 'POWERED_ON':
+                oneandone_conn.modify_server_status(
+                    server_id=server['id'],
+                    action='POWER_OFF',
+                    method='SOFTWARE')
+            elif state == 'running' and server['status']['state'] == 'POWERED_OFF':
+                oneandone_conn.modify_server_status(
+                    server_id=server['id'],
+                    action='POWER_ON',
+                    method='SOFTWARE')
+        except Exception as ex:
```

Comment:
Catch all exceptions should be avoided but it is not a "blocker"

Reply:
Hello @resmo 
Thank you for your feedback. Unfortunately, the 1&1 python SDK is currently throwing a generic `Exception` in `modify_server_status` method, which is why we catch it here.